### PR TITLE
#24 command line option for setting sign, accept thresholds

### LINKED ISSUE #24: New Command line option for setting the thresholds of 'SIGN' and 'ACCEPT'
Please make to accept the 'threshold's for `SIGN` and `ACCEPT` in 'cmd/sebak/cmd/node.go'. The threshold is hardcoded at https://github.com/owlchain/sebak/blob/49a885f9e31675079e872aec698ffc0c672f3693/cmd/sebak/cmd/node.go#L269 .

### PR DESCRIPTION
**Pull request by leejames00**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions.
* If this PR is related to github issues, please mention them in descriptions

### Github Issue

Resolves #24 

### Background

command line option for setting thresholds

### Solution

added command options
`--sign-threshold` and `--accept-threshold`

For example, 
```
sebak node \
    --network-id "this-is-test-sebak-network" \
    --endpoint "https://localhost:12345" \
    --tls-key "sebak.key" \
    --tls-cert "sebak.crt" \
    --storage "file:///tmp/db-n0" \
    --secret-seed SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM \
    --validator "GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2,https://localhost:12346" \
    --validator "GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6,https://localhost:12347" \
    --log-level debug \
    -- sign-threshold 50 --accept-threshold 70
```
### Possible Drawbacks

> Please check the assignee comment in file changed

### CODE REVIEW

File: cmd/sebak/cmd/node.go [hunk 1]
```diff
@@ -265,8 +272,9 @@ func runNode() {
 		os.Exit(1)
 	}
 
-	// TODO policy threshold can be set in cmd options
-	policy, _ := sebak.NewDefaultVotingThresholdPolicy(100, 30, 30)
```
Comment (leejames00) [inline cmd/sebak/cmd/node.go]: BTW, why was the default threshold set to 30? @spikeekips

**Comment by Geod24** [comment]
I think this is a really good case for the config file.

**Comment by spikeekips** [comment]
@Goed24 I think which is better, we can support both, cmd options and config :)

File: cmd/sebak/cmd/node.go [hunk 2]
```diff
@@ -88,6 +89,8 @@ var (
 	flagTLSCertFile         string = sebakcommon.GetENVValue("SEBAK_TLS_CERT", "sebak.crt")
 	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "sebak.key")
 	flagValidators          FlagValidators
+	flagSignThreshold       string = sebakcommon.GetENVValue("SEBAK_SIGN_THRESHOLD", "30")
```
Comment (spikeekips) [inline cmd/sebak/cmd/node.go]: The current threshold, `30` was only for testing. Please suggest the proper value.

File: cmd/sebak/cmd/node.go [hunk 3]
```diff
@@ -134,6 +137,8 @@ func init() {
 	nodeCmd.Flags().StringVar(&flagTLSCertFile, "tls-cert", flagTLSCertFile, "tls certificate file")
 	nodeCmd.Flags().StringVar(&flagTLSKeyFile, "tls-key", flagTLSKeyFile, "tls key file")
 	nodeCmd.Flags().Var(&flagValidators, "validator", "set validator: '<public address>,<endpoint url>,<alias>' or <public address>,<endpoint url>")
+	nodeCmd.Flags().StringVar(&flagSignThreshold, "sign-th", flagSignThreshold, "sign threshold")
```
Comment (spikeekips) [inline cmd/sebak/cmd/node.go]: `sign-th` looks ambiguous, how about just using long expression such like `sign-threshold` and `accept-threshold` in below code?

File: cmd/sebak/cmd/node.go [hunk 4]
```diff
@@ -222,13 +227,15 @@ func parseFlagsNode() {
 
 	// print flags
 	parsedFlags := []interface{}{}
-	parsedFlags = append(parsedFlags, "\n\network-id", flagNetworkID)
+	parsedFlags = append(parsedFlags, "\n\tnetwork-id", flagNetworkID)
```
Comment (spikeekips) [inline cmd/sebak/cmd/node.go]: You filled the missing code 👍

File: cmd/sebak/cmd/node.go (hunk 2 above)
Comment (outsideris) [reply cmd/sebak/cmd/node.go]: 60?

Comment (spikeekips) [reply cmd/sebak/cmd/node.go]: @outsideris 60% will be meaningful. The default threshold of stellar is 67%, https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg#L393

File: cmd/sebak/cmd/node.go (hunk 3 above)
Comment (leejames00) [reply cmd/sebak/cmd/node.go]: ok, seems better 👍

**Comment by leejames00** [comment]
@spikeekips can you review again? thanks

**Comment by leejames00** [comment]
I will merge this PR and change the wiki

**Pull request merged**